addons/sale/models/account_invoice.py

```diff
@@ -97,3 +97,8 @@ def _get_invoice_delivery_partner_id(self):
         # OVERRIDE
         self.ensure_one()
         return self.partner_shipping_id.id or super(AccountMove, self)._get_invoice_delivery_partner_id()
+
+    def _is_downpayment(self):
+        # OVERRIDE
+        self.ensure_one()
+        return bool(self.line_ids.sale_line_ids) and all(sale_line.is_downpayment for sale_line in self.line_ids.sale_line_ids)
```

Comment:
@Feyensv fyi

Reply:
FYI i removed the request for review because I already saw it and didn't have anything to say ...